plugins/jobtype/test/azkaban/jobtype/TestHadoopJobUtilsExecutionJar.java

```diff
@@ -0,0 +1,78 @@
+package azkaban.jobtype;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import azkaban.utils.Props;
+
+public class TestHadoopJobUtilsExecutionJar {
```

Comment:
The original code already tests if the target folder doesn't exist.
Added one more test case where the target folder exists, but nothing is in there.